applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.cpp

```diff
@@ -119,12 +127,53 @@ void InterfacePreprocessCondition::GenerateInterfacePart<3>(
 /***********************************************************************************/
 /***********************************************************************************/
 
+std::unordered_map<IndexType, Properties::Pointer> InterfacePreprocessCondition::CreateNewProperties(ModelPart& rOriginPart)
+{
+    const SizeType number_properties = mrMainModelPart.NumberOfProperties();
+
+    std::unordered_map<IndexType, Properties::Pointer> new_properties;
+    // Reorder id Properties
+    IndexType count = 0;
+    for (auto it_prop = mrMainModelPart.PropertiesBegin(); it_prop < mrMainModelPart.PropertiesEnd(); ++it_prop) {
+        ++count;
+        it_prop->SetId(count);
+    }
+
+    // Create a vector with the ids of the old properties
+    const SizeType number_properties_origin = rOriginPart.NumberOfProperties();
+    std::vector<IndexType> index_vector(number_properties_origin);
+    count = 0;
+    for (auto it_prop = rOriginPart.PropertiesBegin(); it_prop < rOriginPart.PropertiesEnd(); ++it_prop) {
+        index_vector[count] = it_prop->Id();
+        ++count;
+    }
+
+    // Copy to the new properties
+    count = 0;
+    for (auto& i_prop : index_vector) {
+        Properties::Pointer p_original_prop = mrMainModelPart.pGetProperties(i_prop);;
+        ++count;
+        Properties::Pointer p_new_prop = mrMainModelPart.pGetProperties(number_properties + count + 1);
+        new_properties.insert({i_prop, p_new_prop});
+
+        // Now we copy (an remove) the properties we have interest
+        CopyProperties(p_original_prop, p_new_prop, FRICTION_COEFFICIENT);
+        CopyProperties(p_original_prop, p_new_prop, THICKNESS, false);
+        CopyProperties(p_original_prop, p_new_prop, YOUNG_MODULUS);
+    }
+
+    return new_properties;
```

Comment:
you are not creating it in the modelpart...

Reply:
In the description I said that this is a PR depending of #1765 (to avoid conflict), this is the same as in that PR